build.gradle

```diff
@@ -52,8 +52,8 @@ ext {
 
 def supportLibVersion = '28.0.0'
 def mockitoVersion = '2.24.5'
-def androidxTestVersion = '1.1.0'
-def espressoVersion = '3.1.0'
+def androidxTestVersion = '1.2.0-beta01'
```

Comment:
No, because the betas are needed for the Robolectric tests to work with the ViewPager used in the Onboarding Screen. [Here's where I learned](https://github.com/robolectric/robolectric/issues/3698#issuecomment-492778693) that I need to use the betas and the PAUSED LooperMode in Robolectric tests.